pkg/kubernetes/manifest/errors.go

```diff
@@ -66,3 +66,14 @@ func (s SampleString) Indent(n int) string {
 	lines := strings.Split(s.String(), "\n")
 	return indent + strings.Join(lines, "\n"+indent)
 }
+
+// ErrorDuplicateName means two resources share the same name using the given
+// nameFormat.
+type ErrorDuplicateName struct {
+	name   string
+	format string
+}
+
+func (e ErrorDuplicateName) Error() string {
+	return fmt.Sprintf("Two resources share the same name '%s'. Please adapt the name template '%s'.", e.name, e.format)
```

Comment:
We lost the link to https://tanka.dev/helm#two-resources-share-the-same-name in the process of refactoring here.

It is in the Helm section of the website, but what's written there also directly applies to Kustomize.

Reply:
yeah, docs can come later imo :)

Reply:
The Helm error message no longer includes the link, which is a small step back, but we can do this properly before release :+1: